content/en/docs/cwgo/getting-started/_index.md

```diff
@@ -0,0 +1,154 @@
+
+## Install
+
+```shell
+$ go install github.com/cloudwego/cwgo@latest
+```
+
+It's easiest to install with the go command, or you can choose to build and install from source yourself. To see where cwgo is installed, use:
+
+```shell
+$ go list -f {{.Target}} github.com/cloudwego/cwgo
+```
+
+To use thrift or protobuf's IDL to generate code, you need to install the corresponding compiler: [thriftgo](https://github.com/cloudwego/thriftgo) or [protoc](https://github.com/protocolbuffers/protobuf/releases).
+
+thriftgo install:
+
+```shell
+$ GO111MODULE=on go install github.com/cloudwego/thriftgo@latest
+```
+
+protoc install
+
+```shell
+# brew install
+$ brew install protobuf
+```
+
+```shell
+# Official image installation, take macos as an example
+$ wget https://github.com/protocolbuffers/protobuf/releases/download/v3.19.4/protoc-3.19.4-osx-x86_64.zip
+$ unzip protoc-3.19.4-osx-x86_64.zip
+$ cp bin/protoc /usr/local/bin/protoc
+# Make sure include/google is placed under /usr/local/include
+$ cp -r include/google /usr/local/include/google
+```
+
+First, we need to install the command-line code generation tools needed to use this example:
+
+1. Make sure the `GOPATH` environment variable has been defined correctly (e.g. `export GOPATH=~/go`) and add `$GOPATH/bin` to the `PATH` environment variable (e.g. `export PATH=$GOPATH/ bin:$PATH`); do not set `GOPATH` to a directory that the current user does not have read and write permissions
+1. Install cwgo: `go install github.com/cloudwego/cwgo@latest`
+1. Install thriftgo: `go install github.com/cloudwego/thriftgo@latest`
+
+After the installation is successful, execute `cwgo --version` and `thriftgo --version` and you should be able to see the output of the specific version number (the version number is different, take x.x.x as an example):
+
+```shell
+$ cwgo --version
+vx.x.x
+
+$ thriftgo --version
+vx.x.x
+
+$ protoc --version
+libprotoc x.x.x
+```
+
+### Determine where to place the code
+
+1. If the code is placed under `$GOPATH/src`, you need to create an additional directory under `$GOPATH/src`, and then get the code after entering this directory:
+
+```shell
+$ mkdir -p $(go env GOPATH)/src/github.com/cloudwego
+$ cd $(go env GOPATH)/src/github.com/cloudwego
+```
+
+2. If you put the code outside GOPATH, you can get it directly
+
+## Precautions
+
+The bottom layer of cwgo uses [kitex](https://www.cloudwego.io/zh/docs/kitex/tutorials/code-gen/code_generation/), [hz](https://www.cloudwego.io/zh/docs/hertz/tutorials/toolkit/toolkit/), [gen](https://gorm.io/gen/index.html) tools, so the corresponding tool specifications also need to be followed, such as [kitex precautions](https://www.cloudwego.io/zh/docs/kitex/tutorials/code-gen/code_generation/#%E4%BD%BF%E7%94%A8-protobuf-idl-%E7%9A%84%E6%B3%A8%E6%84%8F%E4%BA%8B%E9%A1%B9) and [Notes for hz](https://www.cloudwego.io/zh/docs/hertz/tutorials/toolkit/toolkit/#%E6%B3%A8%E6%84%8F%E4%BA%8B%E9%A1%B9).
```

Comment:
Please use the en version of the doc:
1. https://www.cloudwego.io/docs/kitex/tutorials/code-gen/code_generation/
2. https://www.cloudwego.io/docs/hertz/tutorials/toolkit/toolkit/#notes

Reply:
done

Reply:
done